src/routers/cloudinary.ts

```diff
@@ -3,6 +3,67 @@ import { auth } from '../middlewares/auth';
 import { deleteImageCloudinary } from '../services/cloudinaryService';
 
 const router = express();
+/**
+ * @swagger
+ * /cloudinary/image:
+ *   delete:
+ *     tags:
+ *       - Cloudinary operations
+ *     summary: Delete an image from Cloudinary
```

Comment:
summary should contains just - 
"Must be connected as admin" OR
"Must be connected as user"